tests/phpunit/tests/functions/isWpVersionCompatible.php

```diff
@@ -12,14 +12,34 @@ class Tests_Functions_IsWpVersionCompatible extends WP_UnitTestCase {
 	 * Tests is_wp_version_compatible().
 	 *
 	 * @dataProvider data_is_wp_version_compatible
+	 * @runInSeparateProcess
+	 * @preserveGlobalState disabled
```

Comment:
Why are these needed?

Reply:
These shouldn't be needed as there are no constants/static variables at play.

Reply:
Removed